plugins/connectrpc/go/v1.11.0/buf.plugin.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/connectrpc/go
+plugin_version: v1.11.0
+source_url: https://github.com/connectrpc/connect-go
+description: Generates client and server stubs for connect-go. Compatible with the gRPC, gRPC-Web, and Connect RPC protocols.
```

Comment:
Can we s/connect-go/Connect? Or go straight to `connectrpc.com/connect`?

Reply:
Updated to `connectrpc.com/connect`.